Formula/apt.rb

```diff
@@ -1,8 +1,8 @@
 class Apt < Formula
   desc "Advanced Package Tool"
   homepage "https://wiki.debian.org/apt"
```

Comment:
the homepage redirects to here:
```suggestion
  homepage "https://wiki.debian.org/Apt"
```